### DISCUSSION TITLE
Dialog doesn't work on ios < 13

### DISCUSSION THREAD

**Discussion by hoandq-amela** [tailwindlabs/headlessui#2820] [General]
In low-end devices (eg: iphone 6, IOS <= 13) when accessing screens containing components Dialog of this library, the screen will be dead like in the video.
Error: An unexpected error has occurred.
https://github.com/tailwindlabs/headlessui/assets/133618442/2d1dfdae-62af-43d5-8cce-a8e30f09e888

**Comment by Kaka-123-D** [comment]
Same need !
